alpakaConfig.cmake

```diff
@@ -448,6 +463,147 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
+#               ENDIF()
+#
+#                IF(ALPAKA_CUDA_FTZ)
+#                    LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-fcuda-flush-denormals-to-zero")
+#               ENDIF()
+
+#             IF(ALPAKA_CUDA_SHOW_REGISTER)
+#                  LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-Xcuda-ptxas=-v")
+#               ENDIF()
+#
+#              IF(ALPAKA_CUDA_KEEP_FILES)
+#                   LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-save-temps")
+#                ENDIF()
+
+#            ELSE()
+
+                IF(CUDA_VERSION VERSION_EQUAL 8.0)
+                    LIST(APPEND HIP_HIPCC_FLAGS "-Wno-deprecated-gpu-targets")
+                ENDIF()
+
+                IF(NOT CUDA_VERSION VERSION_LESS 7.5)
+                    LIST(APPEND HIP_HIPCC_FLAGS "--expt-extended-lambda")
+                ENDIF()
+
+                LIST(APPEND HIP_HIPCC_FLAGS "-arch=${ALPAKA_HIP_ARCH}")
+
+                IF(NOT MSVC)
+                    LIST(APPEND HIP_HIPCC_FLAGS "-std=c++11")
+#not compiler flag
+                    SET(HIP_HOST_COMPILER "${CMAKE_CXX_COMPILER}")
+                ENDIF()
+
+                if(CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
+                    LIST(APPEND HIP_HIPCC_FLAGS "-g" "-G")
+                ENDIF()
+
+                IF(ALPAKA_HIP_FAST_MATH)
+                    LIST(APPEND HIP_HIPCC_FLAGS "--use_fast_math")
+                ENDIF()
+
+                IF(ALPAKA_HIP_FTZ)
+                    LIST(APPEND HIP_HIPCC_FLAGS "--ftz=true")
+                ELSE()
+                    LIST(APPEND HIP_HIPCC_FLAGS "--ftz=false")
+                ENDIF()
+
+                IF(ALPAKA_HIP_SHOW_REGISTER)
+                    LIST(APPEND HIP_HIPCC_FLAGS "-Xptxas=-v")
+                ENDIF()
+
+                IF(ALPAKA_HIP_KEEP_FILES)
+                    MAKE_DIRECTORY("${PROJECT_BINARY_DIR}/hcc_tmp")
+                    LIST(APPEND HIP_HIPCC_FLAGS "--keep" "--keep-dir" "${PROJECT_BINARY_DIR}/hcc_tmp")
+                ENDIF()
+
+                OPTION(ALPAKA_HIP_SHOW_CODELINES "Show kernel lines in cuda-gdb and cuda-memcheck" OFF)
+                IF(ALPAKA_HIP_SHOW_CODELINES)
+                    LIST(APPEND HIP_HIPCC_FLAGS "--source-in-ptx" "-lineinfo")
+                    IF(NOT MSVC)
+                        LIST(APPEND HIP_HIPCC_FLAGS "-Xcompiler" "-rdynamic")
+                    ENDIF()
+                    SET(ALPAKA_HIP_KEEP_FILES ON CACHE BOOL "activate keep files" FORCE)
+                ENDIF()
+                LIST(APPEND _ALPAKA_LINK_LIBRARIES_PUBLIC "general;${CUDA_CUDART_LIBRARY}")
+                LIST(APPEND _ALPAKA_INCLUDE_DIRECTORIES_PUBLIC ${HIP_INCLUDE_DIRS})
+
+                LIST(APPEND HIP_HIPCC_FLAGS
+                    -I${_ALPAKA_ROOT_DIR}/include/
+                    -I${_ALPAKA_ROOT_DIR}/test/common/include/
+                    #        "-Xcompiler -fopenmp"
+                    )
```

Comment:
I am not sure what this should do. The comment should be removed and the rest can hopefully be removed or replaced.